src/Components/MultipleFields.tsx

```diff
@@ -0,0 +1,68 @@
+import { Stack, H1 } from "@deskpro/app-sdk";
+import { IAzureWorkItem, IAzureWorkItemFields } from "../types/azure/workItem";
+import { splitArrEvery2 } from "../utils/utils";
+import { workItemFields } from "../utils/workItemFields";
+import { TwoColumn } from "./TwoColumn";
+
+type Props = {
+  data: string[];
+  processName: string;
+  workItemTypeName: string;
+  item: IAzureWorkItem;
+  fieldName: string;
+};
+
+export const MultipleFields = ({
+  data,
+  processName,
+  workItemTypeName,
+  item,
+  fieldName,
+}: Props) => {
+  return (
+    <Stack vertical gap={12} style={{ width: "100%" }}>
+      <H1>{fieldName}</H1>
+      {splitArrEvery2(data).map((field, i) => {
+        const fields = workItemFields[processName][workItemTypeName];
+
+        let [leftText, rightText] = field.map(
+          (fieldName) =>
+            item.fields[
+              fields.find((nameAndField) => nameAndField.name === fieldName)
+                ?.field as keyof IAzureWorkItemFields
+            ] as string
+        );
+
+        if (
+          isNaN(Number(leftText)) &&
+          new Date(leftText).toString() !== "Invalid Date"
+        ) {
+          leftText = new Date(leftText).toLocaleDateString();
```

Comment:
you can use a function [`format()`](https://date-fns.org/v2.29.3/docs/format) and `dd MMM, yyyy` pattern to format the date

Reply:
this code `new Date(leftText).toString() !== "Invalid Date"` doesn't inspire confidence

Reply:
![image](https://user-images.githubusercontent.com/54243490/206225481-59d0b4ed-8576-45f2-9102-1b720adbfb55.png)
Unfortunately thats the only decent way of checking if its a date, their isDate method is broke, as this is the code they use and it outputs true

Reply:
They also have the isValid method. Combine it both to correctly work

Reply:
Does your approach work correctly in all browsers?

Reply:
Also doesn't work, as you can just give it a number and it works 
![image](https://user-images.githubusercontent.com/54243490/206229851-b74cbba5-3fa3-4eef-9bc6-1f74b4fca58d.png)

Reply:
Lemme try another browser

Reply:
Seems like it, tested it on firefox and safari.